e2e-tests/cypress/support/selectorsCreateSignal.ts

```diff
@@ -12,7 +12,10 @@ export const CREATE_SIGNAL = {
   dropdownMinuten: '[data-testid=selectMinutes]',
   dropdownSource: '[data-testid="source"]',
   dropdownSubcategory: '[data-testid="subcategory"]',
+  errorDateTime: ':nth-child(3) > p',
```

Comment:
These seems fragile. Can you add a data-testid attribute to the `<Header />` component? Something like 

`<ErrorItem data-testid={`${meta.name}-required`}>`

Reply:
Added data-testid